docs/src/content/docs/it/guides/css-and-tailwind.mdx

```diff
@@ -0,0 +1,289 @@
+
+- `colors.accent` — utilizzato per i collegamenti e per l'evidenziazione dell'elemento corrente
+- `colors.gray`: utilizzato per i colori e i bordi dello sfondo
+- `fontFamily.sans`: utilizzato per l'interfaccia utente e il testo del contenuto
+- `fontFamily.mono` — utilizzato per esempi di codice
+
+```js {12,14,18,20}
+// tailwind.config.cjs
+const starlightPlugin = require('@astrojs/starlight-tailwind');
+const colors = require('tailwindcss/colors');
+
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+	theme: {
+		extend: {
+			colors: {
+				// Il tuo colore preferito. Indigo è il più vicino alle impostazioni predefinite di Starlight.
+				accent: colors.indigo,
+				// La tua scala di grigi preferita. Lo zinco è il più vicino ai valori predefiniti di Starlight.
+				gray: colors.zinc,
+			},
+			fontFamily: {
+				sans: ['"Atkinson Iperleggibile"'],
+				// Il tuo carattere di testo preferito. Starlight utilizza uno stack di caratteri di sistema per impostazione predefinita.
+				sans: ['"Atkinson Hyperlegible"'],
+				// Il tuo carattere di codice preferito. Starlight utilizza i caratteri di sistema a spaziatura fissa per impostazione predefinita.
+				mono: ['"IBM Plex Mono"'],
+			},
+		},
+	},
+	plugins: [starlightPlugin()],
+};
+```
+
+## Temi
+
+Il tema colore di Starlight può essere controllato sovrascrivendo le sue proprietà personalizzate predefinite.
+Queste variabili vengono utilizzate in tutta l'interfaccia utente con una gamma di sfumature di grigio utilizzate per i colori del testo e dello sfondo e un colore in risalto utilizzato per i collegamenti e per evidenziare gli elementi correnti nella navigazione.
+
+### Editor di temi colore
+
+Utilizza i dispositivi di scorrimento qui sotto per modificare l'accento di Starlight e le tavolozze dei colori grigi.
+Le aree di anteprima scure e chiare mostreranno i colori risultanti e anche l'intera pagina verrà aggiornata per visualizzare in anteprima le modifiche.
+
+Quando sei soddisfatto delle modifiche, copia il codice CSS o Tailwind di seguito e utilizzalo nel tuo progetto.
+
+import ThemeDesigner from '../../../../components/theme-designer.astro';
+
+<ThemeDesigner
+	labels={{
+		presets: {
+			label: 'Preimpostazioni'
+			ocean: 'Oceano'
+			forest: 'Foresta'
+			oxide: 'Ossido'
+			nebula: 'Nebulosa'
+			default: 'Predefinito'
+			random: 'Casuale'
+		},
+		editor: {
+			accentColor: 'Accento'
+			grayColor: 'Grigio'
+			hue: 'Tonalità'
+			chroma: 'Croma'
+			pickColor: 'Scegli colore'
+		},
+		preview: {
+			darkMode: 'Modalità oscura'
+			lightMode: 'Modalità luce'
+			bodyText:
+				'Il corpo del testo viene visualizzato in tonalità grigia con un contrasto elevato con lo sfondo.'
+			linkText: 'I collegamenti sono colorati.'
+			dimText: 'Alcuni testi, come il sommario, hanno un contrasto inferiore.'
+			inlineCode: 'Il codice in linea ha uno sfondo distinto.'
```

Comment:
I think this may be failing due to the missing trailing commas.

```suggestion
			label: 'Preimpostazioni',
			ocean: 'Oceano',
			forest: 'Foresta',
			oxide: 'Ossido',
			nebula: 'Nebulosa',
			default: 'Predefinito',
			random: 'Casuale',
		},
		editor: {
			accentColor: 'Accento',
			grayColor: 'Grigio',
			hue: 'Tonalità',
			chroma: 'Croma',
			pickColor: 'Scegli colore',
		},
		preview: {
			darkMode: 'Modalità oscura',
			lightMode: 'Modalità luce',
			bodyText:
				'Il corpo del testo viene visualizzato in tonalità grigia con un contrasto elevato con lo sfondo.',
			linkText: 'I collegamenti sono colorati.',
			dimText: 'Alcuni testi, come il sommario, hanno un contrasto inferiore.',
			inlineCode: 'Il codice in linea ha uno sfondo distinto.',
```

Reply:
You are definitely right, that might be. Merged as it's a mistake I have to fix anyway. Let's see...

Reply:
Yep, this fixes it.